ci/before_deploy.sh

```diff
@@ -6,20 +6,26 @@ set -ex
 
 # Generate artifacts for release
 mk_artifacts() {
-    RUSTFLAGS="-C target-feature=+ssse3" \
-      cargo build --target $TARGET --release --features simd-accel
+    is_ssse3_target
+    if [ $? -eq 1 ]; then
```

Comment:
Related to my other comment: I don't believe this construction will work reliably anyway — if `is_ssse3_target` returns `1` here, it will cause the script to bail because of `set -e`. The effect of `set -e` needs to be suppressed by putting it in the `if` condition (or using `||` or `&&`)